### DISCUSSION TITLE
Why Mac version requires password for install?

### DISCUSSION THREAD

**Discussion by evrenesat** [aclap-dev/video-downloadhelper#1559] [Help and Q&A]
### Your question

Thanks for maintaining this solution. I haven't installed it yet. Being recommended by Mozilla is a green light but I still wonder why the Mac version of coapp requires a pkg installation with my password? Way it's not an ordinary app in a dmg file?

**Comment by jcfrog** [comment]
Hi,
I can't tell you precisely why we need it but I can tell you it's safe and the code is open source : https://github.com/aclap-dev/vdhcoapp/
